db_stress_tool/db_stress_test_base.cc

```diff
@@ -604,6 +604,32 @@ void StressTest::OperateDb(ThreadState* thread) {
         }
       }
 
+      // Every 1 in N do the following to get live files and wal files test
+      // to check if the status are ok.
+      if (thread->rand.OneInOpt(FLAGS_get_live_files_and_wal_files_one_in)) {
+        Status s;
+        std::vector<std::string> live_file;
+        uint64_t manifest_size;
+        s = db_->GetLiveFiles(live_file, &manifest_size);
+        if (!s.ok()) {
+          fprintf(stdout, "Get live files failed: %s\n", s.ToString().c_str());
```

Comment:
I think we need to fail rather than printing to stdout. This may not cause stress test failure.
Consider to do like the FLAGS_pause_background_one_in() case below.

Reply:
@siying  Sure, will change accordingly.

Reply:
Some other code was setting a bad example. I've formally added an item to our stress test improvement spreadsheet: "Make some stdout only "failures" actually fail the test"